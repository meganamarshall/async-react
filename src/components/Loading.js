import React from 'react';

export function Loading() {
  return <h1>Page is Loading. Wait a sec.</h1>;
}
