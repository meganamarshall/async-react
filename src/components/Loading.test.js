import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

describe('loading component', () => {
  it('renders the loading', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
