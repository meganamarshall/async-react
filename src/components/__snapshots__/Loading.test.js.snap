// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading component renders the loading 1`] = `
<h1>
  Page is Loading. Wait a sec.
</h1>
`;
